src/fundus/publishers/de/heise.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from typing import List, Optional
+
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class HeiseParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = XPath(
+            "//article[not(@data-component='TeaserContainer')]//p[@class='a-article-header__lead']"
+        )
+        _subheadline_selector = XPath("//article[not(@data-component='TeaserContainer')]//h3[@class='subheading']")
+        _paragraph_selector = XPath(
```

Comment:
In [this article](https://www.heise.de/news/Red-Hat-marschiert-mit-Volldampf-in-Richtung-KI-9711991.html?wt_mc=rss.red.ho.ho.rdf.beitrag.beitrag&seite=all) the paragraph about the workshop is also extracted.